src/uct/tcp/tcp_iface.c

```diff
@@ -32,9 +32,13 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_nodelay), UCS_CONFIG_TYPE_BOOL},
 
   {"SNDBUF", "64k",
-   "Socket send buffer size.",
+   "Socket send buffer size. 0 - default value defined by system",
```

Comment:
i would expect the default to be 0
also - with MEMUNITS - there is a special constant "auto" to express a default setting

Reply:
good catch. fixed!